recipe/meta.yaml

```diff
@@ -13,30 +13,30 @@ build:
   script: python setup.py install
 
 requirements:
+  host:
+    - python
+    - pip
+
   build:
-    - setuptools
     - python
```

Comment:
Normally it should be enough to have `python` and `pip` in the host section. Why were they needed here again?